website/docs/r/google_service_account_iam.html.markdown

```diff
@@ -0,0 +1,111 @@
+
+~> **Note:** `google_service_account_iam_binding` resources **can be** used in conjunction with `google_service_account_iam_member` resources **only if** they do not grant privilege to the same role.
+
+## google\_service\_account\_iam\_policy
+
+```hcl
+data "google_iam_policy" "admin" {
+  binding {
+    role = "roles/editor"
+
+    members = [
+      "user:jane@example.com",
+    ]
+  }
+}
+
+resource "google_service_account_iam_policy" "admin-account-iam" {
+	service_account_id = "your-service-account-id"
+	policy_data = "${data.google_iam_policy.admin.policy_data}"
+}
+```
+
+## google\_service\_account\_iam\_binding
+
+```hcl
+resource "google_service_account_iam_binding" "admin-account-iam" {
+  service_account_id = "your-service-account-id"
+  role        = "roles/editor"
+
+  members = [
+    "user:jane@example.com",
+  ]
+}
+```
+
+## google\_service\_account\_iam\_member
+
+```hcl
+resource "google_service_account_iam_member" "admin-account-iam" {
+  service_account_id = "your-service-account-id"
+  role        = "roles/editor"
+  member      = "user:jane@example.com"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_account_id` - (Required) The service account id to apply policy to.
+
+* `member/members` - (Required) Identities that will be granted the privilege in `role`.
+  Each entry can have one of the following values:
+  * **allUsers**: A special identifier that represents anyone who is on the internet; with or without a Google account.
+  * **allAuthenticatedUsers**: A special identifier that represents anyone who is authenticated with a Google account or a service account.
+  * **user:{emailid}**: An email address that represents a specific Google account. For example, alice@gmail.com or joe@example.com.
+  * **serviceAccount:{emailid}**: An email address that represents a service account. For example, my-other-app@appspot.gserviceaccount.com.
+  * **group:{emailid}**: An email address that represents a Google group. For example, admins@example.com.
+  * **domain:{domain}**: A Google Apps domain name that represents all the users of that domain. For example, google.com or example.com.
+
+* `role` - (Required) The role that should be applied. Only one
+    `google_service_account_iam_binding` can be used per role.
+
+* `policy_data` - (Required only by `google_service_account_iam_policy`) The policy data generated by
+  a `google_iam_policy` data source.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are
+exported:
+
+* `etag` - (Computed) The etag of the service account IAM policy.
+
+## Import
+
+IAM member imports use space-delimited identifiers; the resource in question, the role, and the member. This member resource can be imported using the `service_account_id`, role, and member e.g.
```

Comment:
If it were me, maybe I'd collapse these three into one "code sample" section and put the explanations above?  I know I didn't do that for the doc I wrote, but now seeing it I think that might be clearer.  Not feeling strongly about it.

Reply:
Done. What do you think? Do you think I should elaborate around the fact that role is only used for member/binding and member only for the member resource? Or does the example speak for itself.